### ISSUE TITLE
Color coding error in the latest version

### ISSUE THREAD

**Issue by architv** [architv/soccer-cli#60]
There's an error in the color coding in the latest version of `soccer-cli` on GitHub.

**PyPi version**
![soccer_colors_right](https://cloud.githubusercontent.com/assets/5270425/13169235/073727c2-d70a-11e5-8498-64dc334a7f4d.png)

**GitHub version**
![soccer_distort_colors](https://cloud.githubusercontent.com/assets/5270425/13169253/18d59798-d70a-11e5-9bae-1781afe22654.png)

**Comment by Saturn** [comment]
The colours are working the same for me.

It looks like the away wins are coloured orange and the home wins are red. Was this change ever merged in by someone?

And both teams should be yellow for a draw. Strange.


You could step through each commit and see when it works again?

**Comment by architv** [comment]
I'll look into this.

**Comment by architv** [comment]
There seems to be no logical error in the code. Also, the colors work fine if the terminal background is black. For colored screens making the scores either both bold or both not bold works.
https://github.com/architv/soccer-cli/blob/master/soccer/writers.py#L200
ie

```
click.secho('%-25s %2s' % (result.homeTeam, result.goalsHomeTeam),
                    bold=True, fg=homeColor, nl=False)
        click.secho("  vs ", nl=False)
        click.secho('%2s %s' % (result.goalsAwayTeam,
                                result.awayTeam.rjust(25)), fg=awayColor,
                    bold=True, nl=add_new_line)
```
@Saturn Should I sent in a fix?

**Comment by Saturn** [comment]
I wonder if the scores would look better not being bold. Maybe it is just my terminal and font but I think I might prefer it that way.

I actually never noticed that the home team results were in bold and away teams weren't.

**Comment by architv** [comment]
Okay I am making both of them not bold for now.

**Comment by architv** [comment]
Fixed with 58b6067
